2024-01-25 13:48:19,078 - process_api - INFO - run schema: "Asset Test"
2024-01-25 13:48:19,078 - process_api - INFO - run process: "main"
2024-01-25 13:48:19,078 - process_api - INFO - run step: "start"
2024-01-25 13:48:19,078 - process_api - INFO - perform navigate: ${state.server}/#dashboard/AssetTree
2024-01-25 13:48:19,094 - process_api - INFO - run step: "wait_for_tree"
2024-01-25 13:48:19,094 - process_api - INFO - wait for attribute: {'query': '#asset-tree', 'attr': 'status', 'value': 'ready'}
2024-01-25 13:48:20,562 - process_api - INFO - run step: "assert_correct_spelling"
2024-01-25 13:48:20,562 - process_api - INFO - assert text_content_eq: {'query': '#pb-1:shadow:h2', 'value': 'Asset Tree'}
2024-01-25 13:48:20,562 - process_api - INFO - wait for text_content: {'query': '#pb-1:shadow:h2', 'value': 'Asset Tree'}
2024-01-25 13:48:20,584 - process_api - ERROR - Message: invalid selector: An invalid or illegal selector was specified
  (Session info: chrome=118.0.5993.70); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#invalid-selector-exception
Stacktrace:
	GetHandleVerifier [0x00007FF73A708EF2+54786]
	(No symbol) [0x00007FF73A675612]
	(No symbol) [0x00007FF73A52A64B]
	(No symbol) [0x00007FF73A52F116]
	(No symbol) [0x00007FF73A53115F]
	(No symbol) [0x00007FF73A531250]
	(No symbol) [0x00007FF73A56B424]
	(No symbol) [0x00007FF73A56B91C]
	(No symbol) [0x00007FF73A5A6D87]
	(No symbol) [0x00007FF73A58BEAF]
	(No symbol) [0x00007FF73A5A4D02]
	(No symbol) [0x00007FF73A58BC43]
	(No symbol) [0x00007FF73A560941]
	(No symbol) [0x00007FF73A561B84]
	GetHandleVerifier [0x00007FF73AA57F52+3524194]
	GetHandleVerifier [0x00007FF73AAAD800+3874576]
	GetHandleVerifier [0x00007FF73AAA5D7F+3843215]
	GetHandleVerifier [0x00007FF73A7A5086+694166]
	(No symbol) [0x00007FF73A680A88]
	(No symbol) [0x00007FF73A67CA94]
	(No symbol) [0x00007FF73A67CBC2]
	(No symbol) [0x00007FF73A66CC83]
	BaseThreadInitThunk [0x00007FFBFFD4257D+29]
	RtlUserThreadStart [0x00007FFC00F8AA58+40]

2024-01-25 13:48:20,587 - process_api - ERROR - Traceback (most recent call last):
  File "C:\PragmaDev\ui_testing_service\process_api\__init__.py", line 82, in run
    return await self.process_runner.run_step(self, step, ctx, process, item, callback)
  File "C:\PragmaDev\ui_testing_service\process_api\process_runner.py", line 32, in run_step
    result = await function(api, step, ctx, process, item)
  File "C:\PragmaDev\ui_testing_service\process_api\modules\selenium\__init__.py", line 56, in get
    return await get(selenium_driver, step["args"])
  File "C:\PragmaDev\ui_testing_service\process_api\modules\selenium\automation\get.py", line 20, in get
    element = await get_element(ctx, query, timeout)
  File "C:\PragmaDev\ui_testing_service\process_api\modules\selenium\automation\get.py", line 53, in get_element
    element = wait.until(element_callback(None, {
  File "C:\PragmaDev\ui_testing_service\venv\lib\site-packages\selenium\webdriver\support\wait.py", line 96, in until
    value = method(self._driver)
  File "C:\PragmaDev\ui_testing_service\process_api\modules\selenium\condition_callbacks\element_callback.py", line 6, in _predicate
    found_element = driver.find_element(By.CSS_SELECTOR, args["query"])
  File "C:\PragmaDev\ui_testing_service\venv\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "C:\PragmaDev\ui_testing_service\venv\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "C:\PragmaDev\ui_testing_service\venv\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.InvalidSelectorException: Message: invalid selector: An invalid or illegal selector was specified
  (Session info: chrome=118.0.5993.70); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#invalid-selector-exception
Stacktrace:
	GetHandleVerifier [0x00007FF73A708EF2+54786]
	(No symbol) [0x00007FF73A675612]
	(No symbol) [0x00007FF73A52A64B]
	(No symbol) [0x00007FF73A52F116]
	(No symbol) [0x00007FF73A53115F]
	(No symbol) [0x00007FF73A531250]
	(No symbol) [0x00007FF73A56B424]
	(No symbol) [0x00007FF73A56B91C]
	(No symbol) [0x00007FF73A5A6D87]
	(No symbol) [0x00007FF73A58BEAF]
	(No symbol) [0x00007FF73A5A4D02]
	(No symbol) [0x00007FF73A58BC43]
	(No symbol) [0x00007FF73A560941]
	(No symbol) [0x00007FF73A561B84]
	GetHandleVerifier [0x00007FF73AA57F52+3524194]
	GetHandleVerifier [0x00007FF73AAAD800+3874576]
	GetHandleVerifier [0x00007FF73AAA5D7F+3843215]
	GetHandleVerifier [0x00007FF73A7A5086+694166]
	(No symbol) [0x00007FF73A680A88]
	(No symbol) [0x00007FF73A67CA94]
	(No symbol) [0x00007FF73A67CBC2]
	(No symbol) [0x00007FF73A66CC83]
	BaseThreadInitThunk [0x00007FFBFFD4257D+29]
	RtlUserThreadStart [0x00007FFC00F8AA58+40]


2024-01-25 13:48:20,603 - process_api - ERROR - 'NoneType' object has no attribute 'get_property'
2024-01-25 13:48:20,603 - process_api - ERROR - Traceback (most recent call last):
  File "C:\PragmaDev\ui_testing_service\process_api\__init__.py", line 82, in run
    return await self.process_runner.run_step(self, step, ctx, process, item, callback)
  File "C:\PragmaDev\ui_testing_service\process_api\process_runner.py", line 53, in run_step
    return await self.run_step(api, next_step, ctx, process, item, callback)
  File "C:\PragmaDev\ui_testing_service\process_api\process_runner.py", line 53, in run_step
    return await self.run_step(api, next_step, ctx, process, item, callback)
  File "C:\PragmaDev\ui_testing_service\process_api\process_runner.py", line 32, in run_step
    result = await function(api, step, ctx, process, item)
  File "C:\PragmaDev\ui_testing_service\process_api\modules\selenium\modules\selenium_assert.py", line 135, in text_content_eq
    await WaitModule.text_content(api, step, ctx, process, item)
  File "C:\PragmaDev\ui_testing_service\process_api\modules\selenium\modules\selenium_wait.py", line 84, in text_content
    await wait_for_element_details(api, step, element_property_callback, ctx, process, item)
  File "C:\PragmaDev\ui_testing_service\process_api\modules\selenium\modules\selenium_wait.py", line 21, in wait_for_element_details
    WebDriverWait(selenium_driver, timeout).until(callback(element, args))
  File "C:\PragmaDev\ui_testing_service\venv\lib\site-packages\selenium\webdriver\support\wait.py", line 96, in until
    value = method(self._driver)
  File "C:\PragmaDev\ui_testing_service\process_api\modules\selenium\condition_callbacks\element_property_callback.py", line 8, in _predicate
    value = element.get_property(prop)
AttributeError: 'NoneType' object has no attribute 'get_property'

2024-01-25 13:48:20,864 - process_api - INFO - Task 59888772-d700-4c5e-8b6a-15b7fcea9021 completed in 00:00:01:525
